openquake/calculators/extract.py

```diff
@@ -380,8 +380,11 @@ def extract_sitecol(dstore, what):
     """
     Extracts the site collection array (not the complete object, otherwise it
     would need to be pickled).
-    Use it as /extract/sitecol
+    Use it as /extract/sitecol?param=vs30
```

Comment:
I guess 'zipcode' should be written here instead of vs30

Reply:
Yes, but zipcode normally is not available, while the vs30 is always there, so it makes a good example.